content/tutorials/drift-detection-and-remediation/index.md

```diff
@@ -0,0 +1,224 @@
+
+Modify your EC2 Instance by going to the EC2 Dashboard, then select the EC2 instance named `webserver` and click on the **Tags** tab. Next, add a tag by clicking the **Manage tags** button. Then click the **Add new tag** button with the key `Description` and value of `Pulumi Drift Detection`. Finally, **Save** the changes.
+
+## Run a Drift Detection
+
+Return to the Pulumi Cloud and your stack, and again choose the **Detect Drift** action and click **Deploy**.
+
+After your run completes you will see a warning icon on the Drift tab, indicating that drift has been detected. You can see a detailed summary of what resources have been updated or deleted, the properties that have changed, when the drift run happened and a link to the Deployment with further details.
+
+<video autoplay loop muted playsinline>
+    <source src="pulumi-drift-detected.mp4" type="video/mp4">
+    Your browser does not support the video tag.
+</video>
+
+## Remediate Drift
+
+Now that you have detected and reviewed your infrastructure drift, you can choose to remediate and overwrite the changes made in your cloud provider with the most recently specified desired state of your Pulumi program.
+
+As the final step of this tutorial select the **Remediate drift** option in the Actions drop down on your stack page to remediate the manual changes. Once the deployment is complete, your infrastructure will be restored to the desired state expressed in your code.
+
+## Automate Drift Detection
+
+In addition to manually triggering drift detection, you can automate this process by scheduling regular drift detection runs using [Deployment Schedules](/docs/pulumi-cloud/deployments/schedules/). This ensures that any unexpected changes in your infrastructure are promptly identified and remediated without manual intervention.
+
+## Clean up your resources
+
+To avoid incurring any unwanted charges, clean up the resources you created:
+
+1. Navigate to your stack in Pulumi Cloud and open the **Actions** menu.
+2. Select the **Destroy** option to delete all resources in this stack, and click the confirmation checkbox.
+3. Delete the stack by navigating to the **Settings** tab and choose **Delete stack** to remove the stack, including all its deployment history and state files from Pulumi Cloud.
+4. Delete the GitHub repository you created.
+
+## Next Steps
+
+In this tutorial, you created an EC2 instance with Pulumi, including a VPC with a public subnet, an internet gateway, and a security group allowing HTTP traffic. Then, you introduced drift by manually adding a tag to your EC2 resource via the AWS console. Finally, you learned how to detect your drifted infrastructure and how to remediate the manual changes.
+
+To learn more about Drift Detection with Pulumi, take a look at the following resources:
+
+- Learn more about [Pulumi Drift Detection](docs/pulumi-cloud/deployments/drift/).
+- Learn more about [Pulumi Deployments](/docs/pulumi-cloud/deployments/).
```

Comment:
I'd probably mention learning more about Pulumi refresh here, since that's what drift detection is based on, and if we choose not to cover setting up the schedule here (which I think we should!), we should mention how to set that up also here.

Reply:
done